docs/references/basic-transactions.md

```diff
@@ -267,16 +267,31 @@ manually `MemoSq4gqABAXKb96qnH8TysNcWxMyWCqXgDLGmfcHr`.
 
 </SolanaCodeGroup>
 
-## How to change compute budget for a transaction
+## How to change compute budget, fee, &amp; priority for a transaction
+Transaction (TX) priority is transaction fee divided by compute units (CU), e.g. 
+fee/CU. By default the compute budget is 1.4M CU and the Base Fee is 5,000 
```

Comment:
> Transaction (TX) priority is transaction fee divided by compute units (CU), e.g. fee/CU.

To be clear, transactions are prioritized by ComputeUnitPrice - the micro-lamports value set up `set_compute_unit_price()` instruction. It is _not_ total-transaction-fee / compute-unit-limits.

Transactions willing to pay more per CU are prioritized. The additional fee (aka [prioritization fee](https://github.com/solana-labs/solana/blob/master/docs/src/terminology.md#prioritization-fee)) = compute-unit-limit * compute-unit-price, is added into transaction fee.

>  By default the compute budget is 1.4M CU 

Think this is true: "By default the compute budget is set the product of 200k compute units * number of instructions, with a max of 1.4M compute units. "